src/components/PineTypes/CaseInsensitiveText.js

```diff
@@ -28,6 +28,8 @@ export const rules = {
 
 export const validate = isString
 
+export const normalize = value => value.toString()
```

Comment:
Would it make sense to we make it safe to run against null/undefined values (here and below)?

Reply:
yes indeed, I'll add tests for null and undefined to each of the types